src/tests/unittests/validation/RenderPassValidationTests.cpp

```diff
@@ -37,6 +37,63 @@ TEST_F(RenderPassValidationTest, OneSubpassOneAttachment) {
         .SetSubpassCount(1)
         .SetAttachmentCount(1)
         .AttachmentSetFormat(0, nxt::TextureFormat::R8G8B8A8Unorm)
+        // without a load op
         .SubpassSetColorAttachment(0, 0, 0)
         .GetResult();
+
+    AssertWillBeSuccess(device.CreateRenderPassBuilder())
+        .SetSubpassCount(1)
+        .SetAttachmentCount(1)
+        .AttachmentSetFormat(0, nxt::TextureFormat::R8G8B8A8Unorm)
+        // with a load op
+        .AttachmentSetColorLoadOp(0, nxt::LoadOp::Clear)
+        .SubpassSetColorAttachment(0, 0, 0)
+        .GetResult();
+
+    AssertWillBeSuccess(device.CreateRenderPassBuilder())
+        .SetSubpassCount(1)
+        .SetAttachmentCount(1)
+        .AttachmentSetFormat(0, nxt::TextureFormat::R8G8B8A8Unorm)
+        // with a load op of the wrong type - this is okay, just ignored
+        .AttachmentSetDepthStencilLoadOps(0, nxt::LoadOp::Clear, nxt::LoadOp::Clear)
+        .SubpassSetColorAttachment(0, 0, 0)
+        .GetResult();
+}
+
+// Test for attachment slot arguments out of bounds
+TEST_F(RenderPassValidationTest, AttachmentOutOfBounds) {
+    AssertWillBeError(device.CreateRenderPassBuilder())
```

Comment:
Can we have a control case that passes, then tests for each OOB possiblity with comments to explain what we are testing? It makes following what the test does easier.